NorthstarLauncher/NorthstarLauncher.vcxproj

```diff
@@ -85,6 +88,9 @@
       <AdditionalDependencies>shlwapi.lib;kernel32.lib;user32.lib;gdi32.lib;winspool.lib;comdlg32.lib;advapi32.lib;shell32.lib;ole32.lib;oleaut32.lib;uuid.lib;odbc32.lib;odbccp32.lib;%(AdditionalDependencies)</AdditionalDependencies>
       <StackReserveSize>8000000</StackReserveSize>
     </Link>
+    <PostBuildEvent>
+      <Command>copy /Y "$(TargetPath)" "$(SolutionDir)..\..\"</Command>
+    </PostBuildEvent>
```

Comment:
I think you wanna remove that still before merging, no?

Reply:
these should be kept but I'll make them use an env var soontm